packaging/scripts/post-install.sh

```diff
@@ -32,12 +32,22 @@ function restart_service {
 	fi	
 }
 
+function create_logdir {
+	if [[ ! -d $LOG_DIR ]]; then
+		mkdir -p $LOG_DIR
+		chown -R $DAEMON_USER.$DAEMON_GROUP $LOG_DIR
```

Comment:
I believe this should be `chown -R $DAEMON_USER:$DAEMON_GROUP $LOG_DIR` (thus `.` replaced by `:`)?

> chown [OPTION]... [OWNER][:[GROUP]] FILE...

Reply:
Should work with `.` and `:`
https://serverfault.com/a/194299

But to be on safe `:` is a good idea.

Reply:
I've updated my master for both, chirpstack-network-server and chirpstack-application-server